config/v1/types_scheduling.go

```diff
@@ -20,7 +19,10 @@ type Scheduling struct {
 }
 
 type SchedulingSpec struct {
-	// default node selector (I would be happy to see this die....)
+	// PolicyConfigMap is a reference to a ConfigMap containing scheduler policy which has
+	// user specified predicates and priorities.
+	// The namespace for this configmap is openshift-config
+	PolicyConfigMap ConfigMapNameReference `json:"policyConfigMap,omitempty"`
```

Comment:
Put `// +optional` above if this is optional and I assume it is by `omitempty`

Reply:
Done @soltysh